coralproject/talk
# Ignored User Reply Notifications

## What does this PR do?

Fixes issue raised in https://github.com/coralproject/talk/pull/1488#issuecomment-380064647 by @mkonikov.

## How do I test this PR?

1. Create two user accounts, we'll call them UserA and UserB with reply notifications turned on.
2. Post a comment as UserA and UserB.
3. As UserB, comment on UserA's comment, notice the notification.
4. As UserA, ignore UserB.
5. As UserB, comment on UserA's comment again, notice no notification is sent.
